### ISSUE TITLE
'M_PI' is deprecated: Please use 'Double.pi' or '.pi'

### ISSUE THREAD

**Issue by idevol** [kciter/Floaty#138]
In the file Floaty.swift, on the number line 932

```swift
/**
    Util functions
 */
extension Floaty {
    fileprivate func degreesToRadians(_ degrees: CGFloat) -> CGFloat {
        return degrees / 180.0 * CGFloat(M_PI)
    }
}
```
The constant 'M_PI' is deprecated. Suggests using 'Double.pi' or '.pi' to get the value of correct type and avoid casting.

**Comment by jdrevet** [comment]
Already fixed in commit [#b1c357f](https://github.com/kciter/Floaty/commit/b1c357f3b09abec7b898507b77e3b4b089716dd2) but not yet released

**Comment by kciter** [comment]
Now released. :)
